### ISSUE TITLE
doesn't work at all in windows 10

### ISSUE THREAD

**Issue by linyujing** [icloud-photos-downloader/icloud_photos_downloader#456]
## Overview
[TIP]:  # ( DO NOT include your login-information when pasting in program output! )
[NOTE]: # ( Give a BRIEF summary about your problem )
Installed Python, pip from python website and pip website.
(get warnings when installing pip about the path setting, but it's ok for i'll run under the installation path.)
installed icloudpd.
Get error when run iCloudpd -V
ImportError: cannot import name 'Callable' from 'collections' (C:\Users\myusername\AppData\Local\Programs\Python\Python310\lib\collections\__init__.py)

windows 10 with latest patch;
Python latest stable version for windows 3.10.6
pip 22.2.2

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )

## Context
[NOTE]: # ( Give us any additional information you may have. )

**Comment by andrewcrook** [comment]
@linyujing 

Its a compatability issue with `Python 3.10` I haven't looked to see if its the code or further up the dependency tree.
`Callablable` is no longer in `collections` it's in `collections.abc` because it is an abstract base class. The move started in Python 3.3, if I recall, where is was available in both until now where 3.10 has made the move permanent and removed the old source.  I know this because I have fixed several python projects. I am not sure why but other issues with similar issues have been closed perhaps it has been fixed in a pull request waiting for release? Hopefully, the owner will merge those pull-requests and do a release so we can see where things are at. Otherwise, I may have to fork the project.

~~from collections import Callablable~~
``` python3
from collections.abc import Callablable
```

Meanwhile,  use an earlier version of python or the docker version.
